src/backend/.env

```diff
@@ -0,0 +1,30 @@
+# Chat Service Configuration
+DB_USER=root
```

Comment:
Rename DB_USER=root as MYSQL_ROOT_USER=root and replace it everywhere
REMOVE DB_PASSWORD and use existing MYSQL_ROOT_PASSWORD wherever DB_PASSWORD is used
Remove DB_HOST and make it MYSQL_HOST=mysql_db and replace all its usages

Reply:
I've renamed DB_USER to MYSQL_ROOT_USER, replaced DB_PASSWORD with MYSQL_ROOT_PASSWORD, and updated DB_HOST to MYSQL_HOST.